stdlib/3/unittest/__init__.pyi

```diff
@@ -298,22 +302,30 @@ _TextTestResult = TextTestResult
 
 defaultTestLoader = ...  # type: TestLoader
 
-_ResultClassType = Callable[[TextIO, bool, int], TestResult]
+_ResultClassType = Callable[[IO[str], bool, int], TestResult]
 
 class TestRunner:
     def run(self, test: Union[TestSuite, TestCase]) -> TestResult: ...
 
 class TextTestRunner(TestRunner):
+    stream: IO[str]
+    descriptions: bool
+    verbosity: int
+    failfast: bool
+    buffer: bool
+    tb_locals: bool
+    warnings: Optional[Type[Warning]]
+    resultclass: Optional[_ResultClassType]
```

Comment:
```suggestion
    resultclass: _ResultClassType
```

This is initialized to `TextTestResult` and not overwritten if `resultclass` is `None`.